### PR TITLE
Added video link to  "zh" directory intro.md 

### PR DESCRIPTION
**Pull request by ronyayoung**

### CODE REVIEW

**Review by nibinqtl — CHANGES_REQUESTED** [review]
Please use Chinese for the zh version

**Comment by ronyayoung** [comment]
> Please use Chinese for the zh version

Is there a Chinese version for intro.md under the dev branch?  I saw the getting-started.md is in Chinese and updated the video link there also.

**Comment by nibinqtl** [comment]
> > Please use Chinese for the zh version
> 


> Is there a Chinese version for intro.md under the dev branch? I saw the getting-started.md is in Chinese and updated the video link there also.
All the files under /zh/ need to be translated to Chinese. Any new content should be added in Chinese.
You can start translating some of those files.

**Comment by ronyayoung** [comment]
> > > Please use Chinese for the zh version
> 
> > Is there a Chinese version for intro.md under the dev branch? I saw the getting-started.md is in Chinese and updated the video link there also.
> > All the files under /zh/ need to be translated to Chinese. Any new content should be added in Chinese.
> > You can start translating some of those files.

Ok, I will create a task and translate some of the files in the /zh/ directory, starting from the intro.md.

File: zh/docs/intro.md [hunk 1]
```diff
@@ -36,5 +36,5 @@ The CDN360 content delivery platform provides a rich [API](</apidocs>) that supp
 
 ## Where to Go from Here
 
-Perform the steps in the [CDN360 Quick Start](</docs/getting-started.md>).
+Check the [Getting Started with CDN360 Video](<https://www.cdnetworks.com/wp-content/uploads/videos/cdn360_v1.mp4>) and perform the steps in the [CDN360 Quick Start](</docs/getting-started.md>).
```
Comment (nibinqtl) [inline zh/docs/intro.md:39]: Please change this sentence into Chinese, so it does not need to be translated later.

**Comment by nibinqtl** [comment]
> > > > Please use Chinese for the zh version
> > 
> > 
> > > Is there a Chinese version for intro.md under the dev branch? I saw the getting-started.md is in Chinese and updated the video link there also.
> > > All the files under /zh/ need to be translated to Chinese. Any new content should be added in Chinese.
> > > You can start translating some of those files.
> 
> Ok, I will create a task and translate some of the files in the /zh/ directory, starting from the intro.md.

Please coordinate with Boxiong. Thanks!

**Comment by ronyayoung** [comment]
> > > > > Please use Chinese for the zh version
> > > 
> > > 
> > > > Is there a Chinese version for intro.md under the dev branch? I saw the getting-started.md is in Chinese and updated the video link there also.
> > > > All the files under /zh/ need to be translated to Chinese. Any new content should be added in Chinese.
> > > > You can start translating some of those files.
> > 
> > 
> > Ok, I will create a task and translate some of the files in the /zh/ directory, starting from the intro.md.
> 
> Please coordinate with Boxiong. Thanks!

Sure, will do.

**Review by nibinqtl — APPROVED** [review]

**Pull request merged**